### ISSUE TITLE
Feature Request: accessibility support

### ISSUE THREAD

**Issue by tiamed** [santomegonzalo/react-native-floating-action#208]
Current behavior: Talkback cannot recognize the floating action button.

Expected behavior: Support accessibility label so that talkback can recognize the button.
